bitbots_odometry/src/odometry_fuser.cpp

```diff
@@ -53,6 +68,11 @@ OdometryFuser::OdometryFuser() {
   // so we do not immediately blast something the log output
   ros::Duration(0.5).sleep();
 
+  // wait for transforms from joints
+  while (!tf_buffer_.canTransform("l_sole", "base_link", ros::Time(0), ros::Duration(1)) && ros::ok()) {
+    ROS_WARN("Wainting for transforms from robot joints");
```

Comment:
Maybe this warning could be throttled.

Reply:
it is throttled to 1Hz due to the duration of the canTransform timeout